packages/dds/matrix/src/permutationvector.ts

```diff
@@ -153,8 +153,8 @@ export class PermutationVector extends Client {
             newMergeTreeSnapshotFormat: true,   // Temporarily force new snapshot format until it is the default.
         });                                     // (See https://github.com/microsoft/FluidFramework/issues/84)
 
-        this.mergeTreeDeltaCallback = this.onDelta;
-        this.mergeTreeMaintenanceCallback = this.onMaintenance;
+        this.on("delta", (opArgs, deltaArgs) => this.onDelta);
```

Comment:
you need to pass the parameters to the function in the listener

Reply:
I just removed the arguments since they are already included in the definition of listener, is that okay? I did the same way in `revertible.spec.ts` and currently the test results look good 
![image](https://user-images.githubusercontent.com/86381878/202324385-eceb97ca-17a3-42c2-abe4-e195b7ed02d0.png)